src/routes/(app)/admin[[hash=admin_hash]]/form/new/+page.server.ts

```diff
@@ -27,7 +27,7 @@ export const actions: Actions = {
 				title: z.string().trim().min(1).max(MAX_NAME_LIMIT),
 				target: z.string().trim().min(1).max(100),
 				subject: z.string().trim().min(1).max(100),
-				content: z.string().trim().min(1).max(MAX_CONTENT_LIMIT),
+				content: z.string().trim().min(1).max(MAX_CONTENT_LIMIT).default(''),
```

Comment:
```suggestion
				content: z.string().trim().max(MAX_CONTENT_LIMIT),
```

This is probably the fix